data/developerhub/applications.json

```diff
@@ -155,6 +155,19 @@
       "complexity": ["intermediate"],
       "pro": true,
       "cloudPods": false
+    },
+    {
+      "title": "Messaging Processing application with SQS, DynamoDB, and Fargate",
+      "description": "Messaging Processing application that demonstrates a Fargate container that interacts with SQS & DynamoDB, deployed using Cloud Development Kit on LocalStack",
+      "url": "https://github.com/localstack/sqs-fargate-ddb-cdk-go",
+      "teaser": "https://raw.githubusercontent.com/localstack/sqs-fargate-ddb-cdk-go/main/images/architecture-diagram.png",
+      "services": ["sqs", "ddb", "fargate"],
+      "deployment":  ["cdk"],
+      "platform": ["go"],
```

Comment:
seems like "go" should also be added [here in the platforms.json](https://github.com/localstack/docs/blob/main/data/developerhub/platforms.json)? 🙏